tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -148,6 +149,18 @@ describe('Home - Game List', () => {
 
       // Test that join button is now disabled
       cy.contains('[data-cy-join-game]', 'Play').should('be.disabled');
+
+      // Manually re-enable join button to confirm backend rejects request
+      cy.window().its('cuttle.app.config.globalProperties.$store')
+        .invoke('commit', 'updateGameStatus', {id: gameData.gameId, newStatus: true});
```

Comment:
I spent a while trying to figure out how to test this issue. I tried having the test directly 
```
.invoke('dispatch', 'requestSubscribe', {id: gameData.gameId})
```

But I had trouble managing the promises to make the test pass if we got the expected error and fail otherwise. Plus that doesn't confirm that the snackbar shows up displaying the error message. I thought about disconnecting and reconnecting the socket so the player misses that the game is over, but that's a bit brittle because eventually we'd like the socket reconnect logic to rehydrate the list of which games are available.

Eventually I settled on having the test code manually switch a game to `status: true` by invoking the store mutation directly here. I'm really pleased with how this turned out because it lets the test control the game looking 'available' on the clientside without weird backend implications and then the actual interactions and assertions (clicking the button then asserting the button disables and the snackbar appears) happen organically through the app as usual